binaries/apiserver/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 	level, err := log.ParseLevel(*logLevelFlag)
 	if err != nil {
 		log.Error(err)
-		return
+		level = log.InfoLevel
```

Comment:
Reverting. And it does Exit(1), 
https://github.com/sirupsen/logrus/blob/master/entry.go#L164
https://github.com/sirupsen/logrus/blob/547e984ad93a90192134fc0fdb57a468edb514fe/alt_exit.go#L49